iotbx/mtz/object.cpp

```diff
@@ -246,6 +247,166 @@ namespace iotbx { namespace mtz {
     return result;
   }
 
+  void
+  object::add_dials_batches(int dataset_id,
```

Comment:
Does the name _really_ need to have "dials" in?

Reply:
We thought this to be sensible as this function takes away a lot of the options for setting values and replaces them with DIALS-specific defaults.

Reply:
Also - if DIALS ever needs to write different things we'd want to be free to modify the function signature, so really don't want anyone else to rely on this function.

Reply:
cf https://github.com/dials/dials/pull/936